addons/binding/org.openhab.binding.freebox/src/main/java/org/openhab/binding/freebox/handler/FreeboxHandler.java

```diff
@@ -203,63 +204,100 @@ public void initialize() {
         public void run() {
             logger.debug("Authorize job...");
 
-            String apiBaseUrl = null;
-            String apiVersion = null;
-            String hardwareVersion = null;
-            // String apiDomain = null;
-            // boolean httpsAvailable = false;
-            // String httpsPort = null;
             FreeboxServerConfiguration configuration = getConfigAs(FreeboxServerConfiguration.class);
             String result = null;
-            try {
-                result = HttpUtil.executeUrl("GET", "http://" + configuration.fqdn + "/api_version", 5000);
-            } catch (IOException e) {
-                logger.debug("Cann't connect to Freebox server {}", e);
+            boolean httpsRequestOk = false;
+            if (!Boolean.TRUE.equals(configuration.useOnlyHttp)) {
+                try {
+                    result = HttpUtil.executeUrl("GET", "https://" + configuration.fqdn + "/api_version", 5000);
+                    httpsRequestOk = true;
+                } catch (IOException e) {
+                    logger.debug("Can't connect to {} with HTTPS: {}", configuration.fqdn, e);
```

Comment:
Missed this one, the exception does not need an {}. So there should be one place holder only